src/test/java/spring/jpa/board/controller/PostRestControllerTest.java

```diff
@@ -0,0 +1,221 @@
+package spring.jpa.board.controller;
+
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javassist.NotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.payload.JsonFieldType;
+import org.springframework.test.web.servlet.MockMvc;
+import spring.jpa.board.domain.User;
+import spring.jpa.board.dto.post.PostCreateRequest;
+import spring.jpa.board.dto.post.PostModifyRequest;
+import spring.jpa.board.dto.user.UserConverter;
+import spring.jpa.board.dto.user.UserDto;
+import spring.jpa.board.service.PostService;
+import spring.jpa.board.service.UserService;
+
+@Slf4j
+@AutoConfigureRestDocs
+@AutoConfigureMockMvc
+@SpringBootTest
```

Comment:
https://github.com/prgrms-be-devcourse/SpringBoot-Board/pull/7#discussion_r735049997